src/main/java/com/aws/greengrass/clientdevices/auth/metrics/ClientDeviceAuthMetrics.java

```diff
@@ -33,6 +35,10 @@ public class ClientDeviceAuthMetrics {
             "VerifyClientDeviceIdentity.Success";
     public static final String METRIC_VERIFY_CLIENT_DEVICE_IDENTITY_FAILURE =
             "VerifyClientDeviceIdentity.Failure";
+    public static final String METRIC_AUTHORIZE_CLIENT_DEVICE_ACTIONS_SUCCESS =
```

Comment:
Can we make these private? That would be a good litmus test to see if we actually have the encapsulation that I currently think we do

Reply:
I have them as public so we can access them in the tests as well, but that's the only other place they're used

Reply:
One downside of re-using string constants in tests, is that you can't test that the string itself is what you want. If someone accidentally fat fingered this string, then we'd start emitting a different metric but the tests would still fail. Something to consider.

For things that are only used in tests, consider making them package private

Reply:
Interesting, hadn't thought of that. Would it be better to have them private and write out the string in the tests to ensure they're what we're looking for instead of having them package private?